Back out "fix view listing in autograd codegen"

chenyangyu1988 opened the pull request:
Summary: Original commit changeset: 5ebc4c978af5

Test Plan: backout

Differential Revision: D19588328

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit c91fcdf2:


* **1/2** failures introduced in this PR
* **1/2** recognized as flaky :snowflake:
    * Re-run these jobs?


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=c91fcdf2d95ebfb820d269da93ef1285fae1ae95).

### :detective: 1 new failure recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4353000) pytorch_linux_backward_compatibility_check_test (1/1)
**Step:** "Test" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=91547548) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=91547548))
<details>
<summary>
<code>Jan 27 23:41:18 The PR is introducing backward incompatible changes to the operator library. Please contact PyTorch team to confirm whether this change is wanted or not.</code>
</summary>

```
Jan 27 23:41:18 processing existing schema:  aten::values(Dict(str, t) self) -> (t[](*)) 
Jan 27 23:41:18 processing existing schema:  aten::values(Dict(int, t) self) -> (t[](*)) 
Jan 27 23:41:18 processing existing schema:  aten::values(Dict(float, t) self) -> (t[](*)) 
Jan 27 23:41:18 processing existing schema:  aten::values(Dict(Tensor, t) self) -> (t[](*)) 
Jan 27 23:41:18 processing existing schema:  aten::view(Tensor(a) self, int[] size) -> (Tensor(a)) 
Jan 27 23:41:18 processing existing schema:  aten::where(Tensor condition) -> (Tensor[]) 
Jan 27 23:41:18 processing existing schema:  aten::where.self(Tensor condition, Tensor self, Tensor other) -> (Tensor) 
Jan 27 23:41:18 processing existing schema:  aten::zeros_like(Tensor self, *, int? memory_format=None) -> (Tensor) 
Jan 27 23:41:18 processing existing schema:  aten::zeros_like.dtype(Tensor self, *, int dtype, int layout, Device device, bool pin_memory=False, int? memory_format=None) -> (Tensor) 
Jan 27 23:41:18 processing existing schema:  prim::Constant() -> (...) 
Jan 27 23:41:18 The PR is introducing backward incompatible changes to the operator library. Please contact PyTorch team to confirm whether this change is wanted or not.  
Jan 27 23:41:18  
Jan 27 23:41:18 Broken ops: [ 
Jan 27 23:41:18 	aten::split_with_sizes(Tensor(a) self, int[] split_sizes, int dim=0) -> (Tensor[]) 
Jan 27 23:41:18 ] 
Jan 27 23:41:18 + cleanup 
Jan 27 23:41:18 + retcode=1 
Jan 27 23:41:18 + set +x 
Jan 27 23:41:18 =================== sccache compilation log =================== 
Jan 27 23:41:18 =========== If your build fails, please take a look at the log above for possible reasons =========== 
Jan 27 23:41:18 Compile requests                 0 
```

</details>

### :snowflake: 1 failure recognized as flaky
The following build failures have been detected as flaky and **may not be your fault**:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4351944) pytorch_macos_10_13_py3_build (1/1)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=91495768) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=91495768)) :snowflake:
<details>
<summary>
<code>Jan 27 14:48:39 CondaHTTPError: HTTP 404 NOT FOUND for url <https://repo.anaconda.com/pkgs/main/osx-64/setuptools-45.1.0-py37_0.conda></code>
</summary>

```
Jan 27 14:48:39   certifi                                  2019.9.11-py37_0 --> 2019.11.28-py37_0 
Jan 27 14:48:39   cffi                                1.13.0-py37hb5b8e2f_0 --> 1.13.2-py37hb5b8e2f_0 
Jan 27 14:48:39   conda                                       4.7.12-py37_0 --> 4.8.1-py37_0 
Jan 27 14:48:39   setuptools                                  41.4.0-py37_0 --> 45.1.0-py37_0 
Jan 27 14:48:39  
Jan 27 14:48:39  
Jan 27 14:48:39  
Jan 27 14:48:39 Downloading and Extracting Packages 
Jan 27 14:48:39  setuptools-45.1.0    | 511 KB    | :   0% 0/1 [00:00<?, ?it/s] 
Jan 27 14:48:39  
Jan 27 14:48:39 CondaHTTPError: HTTP 404 NOT FOUND for url <https://repo.anaconda.com/pkgs/main/osx-64/setuptools-45.1.0-py37_0.conda> 
Jan 27 14:48:39 Elapsed: 00:00.065815 
Jan 27 14:48:39 CF-RAY: 55be468108ba71c5-ORD 
Jan 27 14:48:39  
Jan 27 14:48:39 An HTTP error occurred when trying to retrieve this URL. 
Jan 27 14:48:39 HTTP errors are often intermittent, and a simple retry will get you on your way. 
Jan 27 14:48:39  
Jan 27 14:48:39  
Jan 27 14:48:39 + cleanup 
Jan 27 14:48:39 + retcode=1 
Jan 27 14:48:39 + set +x 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

chenyangyu1988 committed:
Back out "Make autogen functions correct for multiple outputs and views"

fbshipit-source-id: 3a9bf4813f1d059be6095f8651fc45a2556c11b8

chenyangyu1988 committed:
Back out "fix view listing in autograd codegen" (#32668)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/32668

Original commit changeset: 5ebc4c978af5

Test Plan: backout

Differential Revision: D19588328

fbshipit-source-id: 8a2fd4bd0ceaa0addf633e219e1e54e83044c9ec

albanD reviewed (changes requested):
This should be closed no?
